scout/adapter/mongo/case.py

```diff
@@ -935,12 +935,17 @@ def _load_clinical_variants(self, case_obj: dict, build: str, update: bool = Fal
             for key, value in ORDERED_FILE_TYPE_MAP.items()
             if value["variant_type"] != "research"
         )
-        load_type_cat = set()
+
+        load_type_cat = []
```

Comment:
Somebody created it! https://github.com/rspeer/ordered-set

Reply:
I could imagine. And no. 😆 We will not have it. Even if it goes 🥺.